Signal/src/ViewControllers/MediaGalleryViewController.swift

```diff
@@ -31,6 +31,10 @@ public class MediaGalleryItem: Equatable, Hashable {
         return attachmentStream.isImage
     }
 
+    var caption: String? {
+        return attachmentStream.caption
```

Comment:
We need to apply `filterStringForDisplay`.  Maybe we should add `[TSAttachment captionForDisplay]`?